docs/workforce-management-framework/topics/pro-running-the-demo-app.adoc

```diff
@@ -1,12 +1,18 @@
-
 [id='{context}-pro-running-the-demo-app']
 = Running the Demo App
```

Comment:
I'm going to add a variable here so you can call them what you want when you/Eng/Product have made a decision on what to call them.